examples/cookbook/testing/unit/counter_app/lib/counter.dart

```diff
@@ -1,7 +1,5 @@
 class Counter {
   int value = 0;
-
```

Comment:
Why are we deleting these blank lines? I believe these add to readability in this instance.

Reply:
I removed them because this did not seem to be consistent with other examples. I'll revert this change.